From 1efff8cd31da8ebce24df4c6b8755cee335bd39e Mon Sep 17 00:00:00 2001
From: Vasyl Gello <vasek.gello@gmail.com>
Date: Tue, 8 Oct 2019 18:53:00 +0000
Subject: [PATCH] Bump Security String to 2019-10-05

Implemented:
============
CVE:            References:     Type:   Severity:       Updated AOSP versions:
CVE-2019-2173   A-123013720     EoP     High            7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2019-2184   A-134578122     RCE     Critical        7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2019-2185   A-136173699     RCE     Critical        7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2019-2186   A-136175447     RCE     Critical        7.1.1, 7.1.2, 8.0, 8.1, 9
CVE-2019-2187   A-124940143     ID      High            7.1.1, 7.1.2, 8.0, 8.1, 9, 10

Not Implemented:
================
CVE:            References:     Type:   Severity:       Updated AOSP versions:
None

Not Applicable (platform source):
=================================
CVE:            References:     Type:   Severity:       Updated AOSP versions:
CVE-2019-2110   A-69703445      ID      High            9
CVE-2019-2114   A-123700348     EoP     High            8.0, 8.1, 9

Change-Id: Ic0c43e5b0fdaca92a58b48abd3a2f9fb2a67af9e
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index f62ca432b5..d82efd6174 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -131,7 +131,7 @@ ifeq "" "$(PLATFORM_SECURITY_PATCH)"
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-    PLATFORM_SECURITY_PATCH := 2019-09-05
+    PLATFORM_SECURITY_PATCH := 2019-10-05
 endif
 
 ifeq "" "$(PLATFORM_BASE_OS)"
-- 
2.17.1

